index.html

```diff
@@ -1037,22 +1037,62 @@
 
 </div>
 
-## Principles for Safety Systems {#safety}
+## Balancing Conflicting Privacy Rights {#balancing}
+
+Sometimes different people's privacy rights come into conflict. Often this happens when
+one user of a service is trying to invade another user's privacy: hiding all private
+information about the first user can prevent the service from protecting the second user.
+
+<div class="practice">
+
+<p><span class="practicelab" id="principle-violation-sacrifices-privacy">When a
+person violates the [=principles=] of a [=context=] they're interacting with, the service
+operating that context may use their data in ways that would otherwise be
+[=inappropriate=] in order to protect other people.</span></p>
+
+The service may also use innocent people's data in order to distinguish them from these
+bad actors.
```

Comment:
I don't think this is intended, but this might read as: bad actors forgo privacy protections by violating the norms of a context and also good actors also forgo those privacy protections if a service may be using that data to protect someone.

That would be a principle that isn't about balance, but that inappropriateness is simply overruled by the purpose of data usage.

Reply:
Good point, I think you've actually read it correctly: the purpose of "preventing users from causing privacy (and other) violations" makes a lot of data collection appropriate (follow the norms/principles of its context) that wouldn't be appropriate for other purposes.

There's still some division between what's appropriate to do with innocent vs guilty users' data: it's probably ok to share guilty users' data across services, while it'd only be ok to use innocent users' data to determine that they're innocent. What would be a better way to say this?